docs/contracts/using-for.rst

```diff
@@ -37,6 +41,50 @@ If you use a list of functions (e.g. ``using {f, g, h, L.t} for uint;``),
 then the type (``uint``) has to be implicitly convertible to the
 first parameter of each of these functions. This check is
 performed even if none of these functions are called.
+Note that private library functions can only be specified when ``using for`` is inside a library.
+
+If you define an operator (e.g. ``using {f as +} for T``), then the type (``T``) must be a
+:ref:`user-defined value type <user-defined-value-types>` and the definition must be a ``pure`` function.
+The following operators can be defined this way:
+
++------------+----------+---------------------------------------------+
+| Category   | Operator | Possible signatures                         |
```

Comment:
```suggestion
| Category   | Operator | Signature                                   |
```

It's the one and only signature of such a function for any given type ``T``, isn't it :-)?

Reply:
There are two possible signatures for `-`.

Reply:
Hm, I'd still think of those as two distinct operators (they're distinguished at the parsing/grammar level fwiw) each with unique signature, but yeah, fair enough, that makes it a bit fuzzy

Reply:
The problem for me is that having two different operators with the same name makes it hard to clearly refer to them in the docs. I find it easier to treat them as the same operator in docs and just say it has multiple signatures or variants. In the end it's just a different way to refer to the same thing anyway.

Reply:
Well, not important enough to hold this up anyways. Even though I'd say it's not the same thing you refer to :-). But let's just keep it as is anyways for now.